types/creativebulma__bulma-tagsinput/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This config requires paths configuraio for scoped package, see:
https://github.com/definitelyTyped/DefinitelyTyped/#what-about-scoped-packages
Probably it was not created by generator